ingestion_server/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: "2.4"
+services:
+    ingestion_server:
+        image: openverse/ingestion_server:1.4.0
+        ports:
+            - "8001:8001"
+        env_file:
+            - .env
+        dns:
+            - 172.31.0.2
```

Comment:
What does this do and why this specific IP? Unless I'm extra-ignorant (which is a very real possibility) a comment explaining this key and value might help.

Reply:
Great call! I'm uncertain of the IP address specifically, but it has to do with assigning a specific DNS server. It was present in the run command prior to my interacting with the ingestion server, and I don't want to remove it for fear of breaking things :sweat_smile:

Reply:
Oooh, I know this! 172.16.0.0–172.31.255.255 is a reserved IP range by the (IANA) for private / internal networking. This specific IP is the default for virtual private networks in AWS. This specific IP is the dns resolution address for a default AWS Virtual Private Cloud.

Reply:
And also the default for DNS resolution in Docker! Which is probably the use case here. I can't find good documentation on this anywhere otherwise I'd share.